README.md

```diff
@@ -126,7 +128,7 @@ Now, we can run the container:
 ```bash
 docker container run --detach --name ipfs-pinner-instance \
        --volume /tmp/data/.ipfs/:/root/.ipfs/  \
-       -p 4001:4001 -p 3001:3001  \
```

Comment:
bitswap removal?

Reply:
yes. It's not used in any form right now. So just removing it to avoid confusion.